styles.css

```diff
@@ -30,13 +30,13 @@ body {
 nav {
     display: flex;
     justify-content: space-evenly;
-    background-color: chocolate;
-    padding: 0.5rem 0;
+
+    background: rgb(231,180,11);
```

Comment:
I think this first background property is redundant, as the second one will just override it